### ISSUE TITLE
Question - Can one enter colours in the form #439a86

### ISSUE THREAD

**Issue by ramblerswebs** [ramblerswebs/Gantry5_Template#8]
When I go into Components > Gantry 5 and try and edit a style colour I don't seem to be able to specify a colour directly
by entering it html value, e.g #439a86
Is that right?
Is it possible to define standard colours like sunset that one could select from?

**Comment by keithgrimes** [comment]
Yes you can enter colours in the form #nnnnnn. When you select to change the colour you also get a colour picker. However, I cannot see a way to define colours by name e.g. mintcake for people to enter. 

You set the colours in the Style page.

**Comment by ramblerswebs** [comment]
I did manage to enter a colour by entering #nnnnnn but it was not easy. I guess this is a Gantry feature.
If the same is true of colours by name would it be an idea to raise an issue with the Gantry team - you never know

**Comment by ramblerswebs** [comment]
PS the problem with not having defined colours will result in people using all sorts of colours.

**Comment by keithgrimes** [comment]
Have raised a ticket with the Gantry team (but not hopeful of a quick resolution). 

The use of the pre-defined styles should limit customisation, but I will not be able to stop any webmaster who wishes to go outside the standard.

Are you ok to have this ticket closed? or keep open to see if the Gantry team respond?
